src/main/java/core/levels/LevelManager.java

```diff
@@ -110,12 +118,50 @@ private void updateEnemies() {
         if (level.getCurrentTime() >= level.getSpawnTime()) {
             Spawner<Character> enemy = enemies.remove(0);
             enemy.spawn();
-            level.setEnemySpawns(enemies);
+            enemySpawns = enemies;
             level.setScore(level.getScore() + 1);
             level.setSpawnTime(level.getSpawnTime() + 15);
         }
     }
 
+    /**
+     * Create HashMap holding levelDifficulty as key, and number of enemies that correspond to
+     * that levelDifficulty, as value
+     *
+     * @return HashMap containing levelDifficulty mapped to number of enemies
+     */
+    private static HashMap<String, Integer> createLevelDifficultyToEnemies() {
+        HashMap<String, Integer> my_dict = new HashMap<>();
+        for (int i=0; i<10; i++){
+            my_dict.put("L" + (i+5), i+5);
+        }
+        return my_dict;
+    }
+
+    /**
+     * Creates list of enemies to be spawned in level
+     * @param numOfEnemies wanted to be spawned in this level
+     * @return enemies list
+     */
+    private List<Spawner<Character>> createEnemyList(int numOfEnemies) {
+        List<Spawner<Character>> enemies = new ArrayList<>();
+        if (level.getCurrentTime() == 0) {
+            for (int i = 0; i < numOfEnemies; i++) {
+                Spawner<Character> enemySpawner = createEnemySpawner();
+                enemySpawner.addSpawnCallback(character -> character.setTeam("enemy"));
+                enemies.add(enemySpawner);
+            }
+        } else{
+            double numEnemies = Math.floor(level.getCurrentTime() / level.getSpawnTime());
+            for (int i = 0; i < numEnemies; i++) {
+                enemies = enemySpawns;
```

Comment:
Why is update enemies not used here?

Reply:
updateEnemies is for spawning the enemy into the world itself, whereas createEnemyList is just to populate the list from which updateEnemies takes enemies to spawn. will make clearer comments for documentation to reflect this